docs/reference-architectures/r.rst

```diff
@@ -0,0 +1,352 @@
+==============
+CrateDB with R
+==============
+
+Abstract
+========
+
+Statistical analysis and visualization on huge datasets is a common task many
+data scientists face in their day to day life. One common tool for doing this is
+`R`_ - a programming language and software environment geared towards statistical
+computing. CrateDB can be paired with R to provide a source of large, real-time
+machine-data for use in statistical analysis, visualization and machine learning.
+
+Similarly, CrateDB can also be used by R as a store of the results of these statistical
+computations.
+
+This can be accomplished with the `RPostgreSQL`_ library.
+
+Implementation
+==============
+
+Set Up
+------
+
+For this implementation, we will be using the classic `iris classification problem`_.
+
+We have details about 3 species of irises: *iris setosa*, *iris veriscolor*, and
+*iris virginica*. These 3 species can be differentiated by the width and length
```

Comment:
Numberings under 10 to be spelt out? https://intranet.crate.io/handbook/conventions/docs/style.html#numbering